extensions/amp-story/amp-story.md

```diff
@@ -1386,6 +1386,177 @@ The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON confi
   ]
 }
 
+```
+## Experimental Feature: Branching
+
+We’ve implemented additional features to allow for the construction of non-linear stories. By utilizing these features, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story.  Example use cases include quizzes, choose-your-own-story experiences, and story table of contents.
+
+Note that all branching features are behind an `amp-story-branching` experiment. To turn on the experiment, enable it in your dev console:
+
+``` AMP.toggleExperiment('amp-story-branching') ```
+
+For more information see, [experimental features](https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html?format=stories).
+
+## Manipulating Navigation
+The features listed below allow you to manipulate navigation within a story. Note that when using branching features, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. That is, if a user navigates from page-1 to page-5 and then page-6, navigating backwards will follow this exact path (page-6 to page-5 and finally page-1). The other pages will not be exposed to the user.
+
+### The `goToPage` Action
```

Comment:
remove "The" and lowercase "A" in "action"

Reply:
Done.